library/std/arithmetic.qs

```diff
@@ -33,6 +34,32 @@ namespace Microsoft.Quantum.Arithmetic {
         adjoint self;
     }
 
+    /// # Summary
+    /// Applies a bitwise-XOR operation between a classical integer and an
+    /// integer represented by a register of qubits.
+    ///
+    /// # Description
+    /// Applies `X` operations to qubits in a little-endian register based on
+    /// 1 bits in an integer.
+    ///
+    /// Let us denote `value` by a and let y be an unsigned integer encoded in `target`,
+    /// then `ApplyXorInPlace` performs an operation given by the following map:
+    /// |y⟩ -> |y ⊕ a⟩, where ⊕ is the bitwise exclusive OR operator.
```

Comment:
```suggestion
    /// |y⟩ ↦ |y ⊕ a⟩, where ⊕ is the bitwise exclusive OR operator.
```
We might want to use this unicode character. We are already using unicode (for XOR) and it's more accurate than ->

Reply:
Updated in both places.